staging/src/k8s.io/client-go/tools/events/eventseries_test.go

```diff
@@ -0,0 +1,208 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package events
+
+import (
+	"strconv"
+	"testing"
+	"time"
+
+	"os"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/events/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	ref "k8s.io/client-go/tools/reference"
+)
+
```

Comment:
I don't want to block this PR for much longer, but could you please add a TODO (and work on it in the follow up) to add tests for:
- refreshExistingSeries
- finishExistingSeries
[obviously if you want to do that in this PR, it would be even better]

Reply:
I can open another PR to increase test coverage for this.

Reply:
Please add a TODO for it (ideally in the implementation file so that we won't forget about it and will be visible), and I'm fine with this PR then.